### PR TITLE
Add the Store settings to storefront

### PR DESCRIPTION
**Pull request by estacioneto**
#### What is the purpose of this pull request?

As the title says...

#### What problem is this solving?

Needed to change the store settings in the Admin Apps, but now the storefront does it for us.

#### How should this be manually tested?

[Access the workspace](https://pwa--storecomponents.myvtex.com/admin/cms/storefront)

#### Screenshots or example usage

image

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/EditorContainer/StoreEditor/Store/index.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import StoreForm from './StoreForm'
+
+interface Props {
+  iframeWindow: Window
+}
+
+const Store: React.FunctionComponent<Props> = ({ iframeWindow }) => {
+  console.log(iframeWindow)
```
Comment (klzns) [inline react/components/EditorContainer/StoreEditor/Store/index.tsx]: 🔫

File: react/components/form/ArrayFieldTemplateItem.tsx [hunk 2]
```diff
@@ -68,7 +70,7 @@ class ArrayFieldTemplateItem extends Component<Props, State> {
           <div className="flex items-center">
             {showDragHandle && <Handle />}
             <label className="f6 accordion-label-title">
-              <SimpleFormattedMessage id={title} />
+              {title ? <SimpleFormattedMessage id={title} /> : 'Item'}
```
Comment (klzns) [inline react/components/form/ArrayFieldTemplateItem.tsx]: Missing intl?

**Comment by klzns** [comment]
I would also remove those helpTexts:
image

Leaving just the last one

File: messages/context.json [hunk 3]
```diff
@@ -111,5 +111,7 @@
   "pages.editor.store.button.settings.title": "pages.editor.store.button.settings.title",
   "pages.editor.store.button.theme.title": "pages.editor.store.button.theme.title",
   "pages.editor.store.button.template.title": "pages.editor.store.button.template.title",
-  "pages.editor.store.button.back.title": "pages.editor.store.button.back.title"
+  "pages.editor.store.button.back.title": "pages.editor.store.button.back.title",
```
Comment (brunojdo) [inline messages/context.json]: Add `Back to editor`

**Comment by estacioneto** [comment]
> I would also remove those helpTexts:
> image
> 
> Leaving just the last one

About this one, @klzns , it gets from the store schema. How do you think is the best way to do this removal? Removing the `description` property from the schema properties...
1. inside the vtex.store schema
2. before passing the schema to the form
3. another way

File: react/components/EditorContainer/StoreEditor/index.tsx [hunk 4]
```diff
@@ -11,13 +12,15 @@ const Mode = ({ editor, mode }: Props) => {
   switch (mode) {
     case 'theme':
       return <Styles iframeWindow={editor.iframeWindow} />
+    case 'settings':
+      return <Store iframeWindow={editor.iframeWindow} />
```
[the remark this answers is not in the document]
Comment (estacioneto) [reply react/components/EditorContainer/StoreEditor/index.tsx]: It will be used later... But I'll remove it since we aren't using it right now

**Pull request merged**